gensim/test/test_logentropy_model.py

```diff
@@ -19,9 +19,42 @@
 
 
 class TestLogEntropyModel(unittest.TestCase):
+    TEST_CORPUS = [[(1, 1.0)], [], [(0, 0.5), (2, 1.0)], []]
+
     def setUp(self):
         self.corpus_small = MmCorpus(datapath('test_corpus_small.mm'))
         self.corpus_ok = MmCorpus(datapath('test_corpus_ok.mm'))
+        self.corpus_empty = [[()]]
+
+    def testGeneratorFail(self):
+        # attempt to create model generator
+        def get_generator(test_corpus=TestLogEntropyModel.TEST_CORPUS):
+            for iter in test_corpus:
```

Comment:
`iter` is a built-in name, not a good name for a variable. Do you mean `document`?

Reply:
Yikes! Will change